packages/vue-components/src/questions/Question.vue

```diff
@@ -267,6 +284,35 @@ export default {
         this.markAsWrong(markAsAnsweredIfWrong);
       }
     },
+    checkMultiBlanksAnswer(markAsAnsweredIfWrong) {
+      let numMatching = 0;
+      for (let i = 0; i < this.answers.length; i += 1) {
+        let ansIsCorrect = false;
+        const ansProps = this.answers[i]._props;
```

Comment:
And just one minor nit in the implementation that jumped out:

Is the use of this private property documented in Vue? I'm not sure if it it would be wise to use an internal representation.
Perhaps we could use the documented `$props` https://v2.vuejs.org/v2/api/#vm-props instead.